styleguide/derek/global-components/tracking/tracking.js

```diff
@@ -181,6 +181,35 @@ Zoolander.Tracking = (function Tracking($) {
         eventNonInteraction: 0,
       });
     });
+
+    $('form.rsFilter-form').on('submit', (e) => {
+      dataLayer.push({
+        event: 'ga.event',
+        eventCategory: 'Resource Center',
+        eventAction: 'Resource Center Filter - Product',
+        eventLabel: $(e.target).find('option:selected').text(),
+        eventValue: '0',
+        eventNonInteraction: 0,
+      });
+
+      dataLayer.push({
+        event: 'ga.event',
+        eventCategory: 'Resource Center',
+        eventAction: 'Resource Center Filter - Type',
+        eventLabel: getCheckboxes($(e.target).find('input:checked')),
+        eventValue: '0',
+        eventNonInteraction: 0,
+      });
+    });
+  }
```

Comment:
Not sure why, but the my changes are not showing up in the www zoolander branch build.
Are there config changes that need to be made there, since moving over tracking?
<img width="608" alt="screen shot 2017-06-27 at 1 47 51 pm" src="https://user-images.githubusercontent.com/799053/27604619-7164390e-5b3f-11e7-8da4-64920e9dac32.png">